engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/advancedGameSetupScreen/AdvancedGameSetupScreen.java

```diff
@@ -640,16 +640,18 @@ private void updateModuleInformation() {
                 if (info == null) {
                     info = ModuleSelectionInfo.remote(remote);
                     modulesLookup.put(remote.getId(), info);
-                    int pos = Collections.binarySearch(sortedModules, info, moduleInfoComparator);
-                    if (pos < 0) { // not yet in the (sorted) list
-                        sortedModules.add(-pos - 1, info); // use "insertion point" to keep the list sorted
-                        allSortedModules.clear();
-                        allSortedModules.addAll(sortedModules);
-                    }
                 }
                 info.setOnlineVersion(remote);
             }
         }
+        sortedModules.clear();
+        allSortedModules.clear();
```

Comment:
You might want to add a comment explaining the difference between `sortedModules` and `allSortedModules` variables or change their names to something better.

Reply:
Hm, these names were used by somebody long time ago and from my perspective they're quite descriptive. 
sortedModules - represent modules for moduleList (UIList widget).
allSortedModules - it's some kind of back-up, contains all sorted modules for cases when we for example reset filter.
I don't think that I can improve it. Maybe you have some ideas?